library/src/main/java/com/proxerme/library/connection/info/entity/EntryCore.java

```diff
@@ -0,0 +1,164 @@
+package com.proxerme.library.connection.info.entity;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.afollestad.bridge.annotations.Body;
+
+/**
+ * Represents the core data of an Entry.
+ *
+ * @author Desnoo
+ */
+public class EntryCore implements Parcelable {
+
+    public static final Creator<EntryCore> CREATOR = new Creator<EntryCore>() {
+        @Override
+        public EntryCore createFromParcel(Parcel in) {
+            return new EntryCore(in);
+        }
+
+        @Override
+        public EntryCore[] newArray(int size) {
+            return new EntryCore[size];
+        }
+    };
+
+    @Body(name = "id")
+    int id;
+
+    @Body(name = "name")
+    String name;
+
+    @Body(name = "genre")
+    String genre;
+
+    @Body(name = "fsk")
+    String fsk;
+
+    @Body(name = "description")
+    String description;
+
+    @Body(name = "medium")
+    String type;
+
+    /**
+     * The overall number of episodes/chapters. It does not include not available episodes.
+     */
```

Comment:
I think it would be better to add JavaDoc for the getters and not for the properties itself. What do you think?